src/views/Predictions/components/RoundCard/OpenRoundCard.tsx

```diff
@@ -172,7 +179,7 @@ const OpenRoundCard: React.FC<OpenRoundCardProps> = ({
               <>
                 <div ref={targetRef}>
                   <Button disabled startIcon={getPositionEnteredIcon()} width="100%" mb="8px">
-                    {t('%position% Entered', { position: positionDisplay })}
+                    {t('%position% Entered', { position: enteredPosition })}
```

Comment:
For null value, what should we show here?

Reply:
Fixed